variables.tf

```diff
@@ -27,20 +27,44 @@ variable "lacework_image" {
   default     = "lacework/datacollector"
 }
 
+variable "lacework_image_pull_policy" {
+  type        = string
+  description = "The pull policy to use for deploying the Lacework datacollector"
+  default     = "IfNotPresent"
+}
+
 variable "namespace" {
   type        = string
   description = "The Kubernetes namespace in which to deploy"
   default     = "default"
 }
 
-variable "pod_cpu" {
+variable "pod_cpu_request" {
   type        = string
   default     = "100m"
-  description = "The amount of CPU units to assign to the Lacework datacollector pod"
+  description = "The amount of CPU units to request for the Lacework datacollector pod"
 }
 
-variable "pod_mem" {
+variable "pod_mem_request" {
   type        = string
   default     = "256Mi"
-  description = "The amount of Memory (Mi) to assign to the Lacework datacollector pod"
+  description = "The amount of Memory to request for the Lacework datacollector pod"
+}
+
+variable "pod_cpu_limit" {
+  type        = string
+  default     = "1"
+  description = "The limit of CPU units for the Lacework datacollector pod"
+}
+
+variable "pod_mem_limit" {
+  type        = string
+  default     = "1024Mi"
+  description = "The limit of Memory for the Lacework datacollector pod"
```

Comment:
@alannix-lw I haven't seen these cpu and mem limits in our defaults, where did you get them from?

It would be good to have the option to add limits but not set them by default.

Reply:
I use this limit. It works well for us.